build/packages/rpm/client/pmm-client.spec

```diff
@@ -164,21 +172,29 @@ fi
 
 %postun
 case "$1" in
-   0) # This is a yum remove.
-      /usr/sbin/userdel pmm-agent
-      %systemd_postun_with_restart pmm-agent.service
-   ;;
    1) # This is a yum upgrade.
       %systemd_postun_with_restart pmm-agent.service
    ;;
 esac
-if [ $1 == 0 ]; then
+if [ $1 -eq 0 ]; then
+  %systemd_postun_with_restart pmm-agent.service
   if /usr/bin/id -g pmm-agent > /dev/null 2>&1; then
     /usr/sbin/userdel pmm-agent > /dev/null 2>&1
     /usr/sbin/groupdel pmm-agent > /dev/null 2>&1 || true
     if [ -f /usr/local/percona/pmm/config/pmm-agent.yaml ]; then
         rm -r /usr/local/percona/pmm/config/pmm-agent.yaml
     fi
+    if [ -f /usr/local/percona/pmm/config/pmm-agent.yaml.bak ]; then
+        rm -r /usr/local/percona/pmm/config/pmm-agent.yaml.bak
+    fi
+    if [ -d /usr/local/percona/pmm/config ] && [ ! "$(ls -A /usr/local/percona/pmm/config)" ]; then
```

Comment:
Does `!` have the same effect as `[ -z "$(ls -A /usr/local/percona/config)" ]`? If it does, then the latter is preferable IMO.

Reply:
Done